MatrixSDK/JSONModels/Event/MXEventRelationThread.h

```diff
@@ -0,0 +1,29 @@
+// 
+// Copyright 2021 The Matrix.org Foundation C.I.C
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#import <MatrixSDK/MatrixSDK.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MXEventRelationThread : MXJSONModel
+
+@property (nonatomic, readonly) MXEvent *latestEvent;
+@property (nonatomic, readonly) NSUInteger numberOfReplies;;
+@property (nonatomic, readonly, getter=isParticipated) BOOL participated;
```

Comment:
Perhaps `hasParticipated` or even `hasCurrentUserParticipated` work better.

Reply:
Chose `hasParticipated`, other one is too long :)